lib/archethic_web/api/jsonrpc/methods/call_contract_function.ex

```diff
@@ -0,0 +1,72 @@
+defmodule ArchethicWeb.API.JsonRPC.Method.CallContractFunction do
+  @moduledoc """
+  JsonRPC method to call a public function
+  """
+
+  alias ArchethicWeb.API.FunctionCallPayload
+  alias ArchethicWeb.API.JsonRPC.Method
+  alias ArchethicWeb.WebUtils
+
+  @behaviour Method
+
+  @spec validate_params(params :: map()) :: {:ok, params :: map()} | {:error, reasons :: list()}
+  def validate_params(params) do
+    case FunctionCallPayload.changeset(params) do
+      %{valid?: true, changes: changed_params} ->
+        {:ok, Map.update(changed_params, :args, [], fn val -> val end)}
+
+      changeset ->
+        reasons =
+          Ecto.Changeset.traverse_errors(changeset, fn
+            {message, [type: {:array, :any}, validation: :cast]} ->
```

Comment:
This would have been better to catch it in `translate_error` function I think, as it any other time we'll use an array it will be already catched